voting/orbs/Elections/helpers_test.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 func TestOrbsVotingContract_initCurrentElection(t *testing.T) {
+	t.Skip()
```

Comment:
why?

Reply:
this tests init with a time base ... not that interesting anymore